server/app.js

```diff
@@ -21,7 +21,7 @@ app.use(logger('dev'));
 
 const port = process.env.PORT || 3000;
 
-app.use(express.static(path.join(__dirname, '/public')));
+// app.use(express.static(path.join(__dirname, '/public')));
```

Comment:
Why do you have this commented out?

Reply:
Eslint flaggeg my html static file. So I decided  to remove the static file and related line of code. I forgot to remove the commented line

Reply:
Please go ahead and delete the commented code since it's not in use

Reply:
Ok